content/fr/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OpenTelemetry
+description: >-
+  Une télémétrie de qualité, universelle et portable pour une observabilité efficace
+outputs:
+  - HTML
+  - REDIRECTS # Include this `content/fr` ONLY
+developer_note:
+  The blocks/cover shortcode (used below) will use as a background image any
+  image file containing "background" in its name.
```

Comment:
Shouldn't this be translated? (it's translated in the spanish version)

Reply:
@dmathieu , I chose to translate only content that will be visible on the website and omit notes or comments. 
@svrnm: what is the best practices for the OpenTelemetry Website localization?

Reply:
can you check with other translation teams, if they do the translations, do them as well. If we are not consistent, raise an issue and share it over at otel-docs-localization slack channel so we can discuss

Reply:
@svrnm : All other translation teams do the translations. I will do them as well.